windows.md

```diff
@@ -397,7 +397,64 @@ With those compatibility things out of the way, you're ready to start the system
     ```
 
     It should print out a welcome message like this:<br><br>
-    <img src="macos-6-docker-hello-world.png"><br><br>
+    <img src="macos-7-docker-hello-world.png"><br><br>
+
+    if opening Docker Desktop or running the command above shows a message that says `Docker Desktop - Windows HyperVisor is not Present`, you may need to enable virtualization on your machine.
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    First check if virtualization is enabled on your computer on the **Performance** tab of the Task Manager (open the Task Manager with <kbd>Ctrl</kbd>-<kbd>Shift</kbd>-<kbd>Escape</kbd>):<br><br>
+
+    <img src="windows-7-virtualisation-enabled.avif"><br><br>
+
+    Continue with the steps below if virtualization is not enabled, Enable virtualization in your BIOS or UEFI:
+
+    1. Restart your computer if it is already started or start it, if not
+    2. Right when the computer is starting up (Still on the black screen befor the computer brand logo blashes), press repeatedly any of these keys: <kbd>Delete</kbd>, <kbd>Esc</kbd>, <kbd>F1</kbd>, <kbd>F2</kbd>, <kbd>F4</kbd>, <kbd>F10</kbd>, or <kbd>F12</kbd>. Each computer manufacturer uses a different key but it may show a brief message at boot time telling you which one to press. If you miss it the first time, restart and try again. If you are not able to enter the BIOS via this method, check this [BIOS key lists](https://www.disk-image.com/faq-bootmenu.htm) for different computers or consult your computers manual
+    3. Press the key next to any option that says `BIOS Setup` or has `Bios`, `UEFI` in it to open BIOS Setup Utility
```

Comment:
I don't have to press any key to open BIOS Setup after pressing `F2` or `Delete` to open BIOS. I have to go to the "Advanced Mode" and this is explained in the next step

Reply:
When your machine is booting, and you press the `F2` or `Delete` button, Is it possible for you to share the screen it takes you to here?

For some computers, it is a screen like this, which is what I also have on my computer and on Anna's computer, which is `Windows 10 Pro`.  on both computers, I needed to press the `F10` key to continue into the BIOS

![startup menu](https://github.com/upleveled/system-setup/assets/74430629/4f8fbe4c-35c8-4eb5-acd6-13e319384df9)

Reply:
I don't know how to share the screen, maybe I can take a video with my phone? 
> For some computers, it is a screen like this, which is what I also have on my computer and on Anna's computer, which is Windows 10 Pro. on both computers, I needed to press the F10 key to continue into the BIOS

This is not the case with my machine. I guess it depends on the Hardware and the BIOS from the different companies. I see the ASUS Logo and press `F2` or `DELETE` and I am instantly in the BIOS.

Reply:
Yes, you can record or take a picture with your phone. I have updated the point to accommodate the concern you raised

dfb059721d3bc87b8bfaff215fbe8248ff90b758